src/test/java/nextstep/member/application/GithubTestController.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.member.application;
+
+import nextstep.member.application.dto.GithubAccessTokenRequest;
+import nextstep.member.application.dto.GithubAccessTokenResponse;
+import nextstep.member.application.dto.GithubProfileResponse;
+import nextstep.utils.GithubResponses;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class GithubTestController {
+    @PostMapping("/github/login/oauth/access_token")
+    public ResponseEntity<GithubAccessTokenResponse> accessToken(
+            @RequestBody final GithubAccessTokenRequest tokenRequest) {
+        if (tokenRequest.getCode().equals(GithubResponses.잘못된_사용자.getCode())) {
```

Comment:
```suggestion
final GithubResponses githubResponse = GithubResponses.findByCode(tokenRequest.getCode());
if (githubResponse == GithubResponses.잘못된_사용자) {
```

이런형태로 해도 가독성이 괜찮을 것 같아요 😄

Reply:
오! 훨씬 좋아 보입니다. 피드백 감사합니다! 😃